### PR TITLE
Add Community Edition section

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+---
+
+## <a name="what-is-tyk-community-edition"></a>What is Tyk Community Edition?
+
+
+Tyk Community Edition is the open-source API Gateway that is developed by our community and supported by our community members and advocates (with some help from the Tyk Technologies team). The Tyk Community Edition is an On-Premises install that consists of:
+
+* [The Tyk API Gateway](https://tyk.io/docs/concepts/tyk-components/gateway/): Tyk's API Gateway that manages your APIs.
+* [The Tyk Pump](https://tyk.io/docs/concepts/tyk-components/pump/): Tyk's analytics data sink, used to send analytics data to platforms such as StatsD, ElasticSearch and InfluxDB.
+* [The Tyk Identity Broker](https://tyk.io/docs/concepts/tyk-components/identity-broker/): Tyk's third-party IDP integration service.
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: I would remove TIB from this tutorial. That's not getting started and confuses the issue.

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+url: "/get-started/with-tyk-community-edition"
+---
+
+## <a name="what-is-tyk-community-edition"></a>What is Tyk Community Edition?
+
+
+Tyk Community Edition is the open-source API Gateway that is developed by our community and supported by our community members and advocates (with some help from the Tyk Technologies team). The Tyk Community Edition is an On-Premises install that consists of:
+
+* [The Tyk API Gateway](https://tyk.io/docs/concepts/tyk-components/gateway/): Tyk's API Gateway that manages your APIs.
+* [The Tyk Pump](https://tyk.io/docs/concepts/tyk-components/pump/): Tyk's analytics data sink, used to send analytics data to platforms such as StatsD, ElasticSearch and InfluxDB.
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: I would remove Tyk Pump from this tutorial - It is not required for Tyk CE

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+*   Set Tyk to listen only on a specific domain for all API traffic.
+*   Set an API to listen on a specific domain (e.g. api1.com, api2.com).
+*   Split APIs over a domain using a path (e.g. api.com/api1, api.com/api2, moreapis.com/api1, moreapis.com/api2 etc).
+*   If you set a hostname for the Gateway, then all non-domain-bound APIs will be on this hostname + the `listen_path`.
+
+## <a name="prerequisites"></a>Prerequisites
+
+*   Ensure port `8080` is open: this is used in this guide for Gateway traffic (the API traffic to be proxied).
+*   You have MongoDB installed and running.
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: MongoDB is not required for Tyk CE

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+*   You have Redis installed and running
+
+## <a name="install-on-ubuntu"></a>Installing on Ubuntu
+
+### Install MongoDb 3.2
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: remove references to Mongo

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+```{.copyWrapper}
+sudo apt-get install -y redis-server
+```
+
+
+
+### Run Installation Scripts
+
+#### Install the Tyk Pump First
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: remove tyk pump

**Review by asoorm — CHANGES_REQUESTED** [review]
I would suggest that we remove any mention of Mongo & Tyk Pump or Tyk Identity Broker from this guide.

**Comment by marksou** [comment]
Updated following review

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md (hunk 4 above)
Comment (asoorm) [reply tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: @marksou ^^

**Comment by buger** [comment]
@asoorm pls check again

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+ * [Manual Instructions](https://packagecloud.io/tyk/tyk-gateway/install#manual-deb)
+ * [Chef](https://packagecloud.io/tyk/tyk-gateway/install#chef)
+ * [Puppet](https://packagecloud.io/tyk/tyk-gateway/install#puppet)
+ * [CI and Build Tools](https://packagecloud.io/tyk/tyk-gateway/ci)
+
+### <a name="configure-tyk-community-edition"></a>Configuring The Gateway 
+
+You can set up the core settings for the Tyk Gateway with a single setup script, however for more involved deployments, you will want to provide your own configuration file. To get things started, run:
+```{.copyWrapper}
+    sudo /opt/tyk-gateway/install/setup.sh --listenport=8080 --redishost=localhost --redisport=6379 --domain=""
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: remove tab for consistency

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+*   `--redisport=6379`: Use port `6379` for Redis.
+*   `--domain=""`: Do not filter domains for the Gateway, see the note on domains below for more about this.
+
+In this example, we don't want Tyk to listen on a single domain. It is recommended to leave the Tyk Gateway domain unbounded for flexibility and ease of deployment.
+
+### Starting Tyk
+
+The Tyk Gateway can be started now that it is configured. Use this commannd to start the Tyk Gateway:
+```{.copyWrapper}
+    sudo service tyk-gateway start
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: remove tab

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+url: "/get-started/with-tyk-community-edition"
+---
+
+## <a name="what-is-tyk-community-edition"></a>What is Tyk Community Edition?
+
+
+Tyk Community Edition is the open-source API Gateway that is developed by our community and supported by our community members and advocates (with some help from the Tyk Technologies team). The Tyk Community Edition is a [Tyk Gateway](https://tyk.io/docs/concepts/tyk-components/gateway/) only edition:
+
+
+#### Pro Tip: Domains with Tyk Gateway
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: Pro Tip just below introduction to `What is Tyk Community Edition`? Maybe move this section to bottom of the page?

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+  main:
+    parent: "Get started"
+weight: 6
+url: "/get-started/with-tyk-community-edition"
+---
+
+## <a name="what-is-tyk-community-edition"></a>What is Tyk Community Edition?
+
+
+Tyk Community Edition is the open-source API Gateway that is developed by our community and supported by our community members and advocates (with some help from the Tyk Technologies team). The Tyk Community Edition is a [Tyk Gateway](https://tyk.io/docs/concepts/tyk-components/gateway/) only edition:
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: Probably a separate issue, but we definitely need to include more info on what Tyk CE is before we dive deep into installation of. Could we create separate issue for this?

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+
+What we've done here is told the setup script that:
+
+*   `--listenport=8080`: Listen on port `8080` for API traffic.
+*   `--redishost=localhost`: Use the hostname `localhost` for Redis.
+*   `--redisport=6379`: Use port `6379` for Redis.
+*   `--domain=""`: Do not filter domains for the Gateway, see the note on domains below for more about this.
+
+In this example, we don't want Tyk to listen on a single domain. It is recommended to leave the Tyk Gateway domain unbounded for flexibility and ease of deployment.
+
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md:69]: Please create new issue/ticket - Need a guide on compiling Tyk CE from source & Also with Docker / Docker Compose.

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+ * [Manual Instructions](https://packagecloud.io/tyk/tyk-gateway/install#manual-rpm)
+ * [Chef](https://packagecloud.io/tyk/tyk-gateway/install#chef)
+ * [Puppet](https://packagecloud.io/tyk/tyk-gateway/install#puppet)
+ * [CI and Build Tools](https://packagecloud.io/tyk/tyk-gateway/ci)
+
+### <a name="configure-tyk-community-edition"></a>Configuring The Gateway 
+
+You can set up the core settings for the Tyk Gateway with a single setup script, however for more involved deployments, you will want to provide your own configuration file. To get things started, run:
+```{.copyWrapper}
+    ssudo /opt/tyk-gateway/install/setup.sh --listenport=8080 --redishost=localhost --redisport=6379 --domain=""
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: `ssudo` should be `sudo`

File: tyk-docs/content/get-started/with-tyk-community-edition/tutorials/tutorials.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+  main:
+    parent: "With Tyk Community Edition"
+    identifier: /with-tyk-community-edition
+weight: 1
+url: "/get-started/with-tyk-community-edition/tutorials"
+---
+
+The following tutorials will help you perform the following tasks:
+
+* Create an API
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/tutorials/tutorials.md]: To add a bit more value, maybe provide direct links to each section of the tutorials for each of the three options.

File: tyk-docs/content/get-started/with-tyk-community-edition/tutorials/create-api.md [hunk 13]
```diff
@@ -3,9 +3,10 @@ date: 2017-03-23T10:16:44Z
 Title: Create an API - Community Edition
 menu:
   main:
-    parent: Community Edition
-    identifier: community-edition-create-api
+    parent: "/with-tyk-community-edition"
+    identifier: /community-edition-create-api
 weight: 1
+url: "/with-tyk-community-edition/tutorials/create-api"
 ---
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/tutorials/create-api.md]: Add `Prerequisites` section to top of this page. Something along lines of: 

> In order to follow along with this tutorial, we will assume that you have successfully installed Tyk CE.

(Providing direct link to install guide)

File: tyk-docs/content/get-started/with-tyk-community-edition/tutorials/create-security-policy.md [hunk 14]
```diff
@@ -3,9 +3,10 @@ date: 2017-03-23T10:39:14Z
...
 menu:
   main:
-    parent: Community Edition
+    parent: "/with-tyk-community-edition"
     identifier: community-edition-create-security-policy
 weight: 3
+url: "/with-tyk-community-edition/tutorials/create-security-policy"
 ---
 
 ## <a name="what-is-a-security-policy"></a>What is a security policy ?
```
Comment (asoorm) [inline tyk-docs/content/get-started/with-tyk-community-edition/tutorials/create-security-policy.md:12]: In this page, please fix/standardise example code spacing. 

1. remove first indentation.
2. 2-space tabs for json
3. I cannot add comments, but there are a few `mmark` files which need updating.

**Review by asoorm — CHANGES_REQUESTED** [review]

**Comment by asoorm** [comment]
@marksou Mostly cleanup suggestions

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md (hunk 10 above)
Comment (marksou) [reply tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md:69]: #486

File: tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md (hunk 9 above)
Comment (marksou) [reply tyk-docs/content/get-started/with-tyk-community-edition/with-tyk-community-edition.md]: #485

**Review by asoorm — APPROVED** [review]

**Pull request merged**